packages/mdc-select/README.md

```diff
@@ -372,6 +372,7 @@ well as interaction targets. See [here](icon/) for more information on using ico
 | `mdc-select__menu` | Mandatory when using the enhanced select. This class should be placed on the `mdc-menu` element within the `mdc-select` element. |
 | `mdc-select__dropdown-icon` | Mandatory. Should be placed on an `i` element within the `mdc-select` element. Used for the dropdown arrow svg and animation.
 | `mdc-select__icon` | Optional. Should be placed on an `i` or `svg` element within the `mdc-select` element. Used for the leading icon.
+| `mdc-select--activated` | Optional. Styles the activated state of select. This class will be added automatically when menu is opened.
```

Comment:
I think the correct term here according to the [spec](https://material.io/design/interaction/states.html#selected) is --selected instead of --activated.

Reply:
I do not see the example of select component using --selected term?

I named this class name after the spec from [Menus](https://material.io/design/components/menus.html):
![image](https://user-images.githubusercontent.com/47312/57890358-72efd280-7805-11e9-807d-ddec26b319ed.png)

Where activated indicates that the select dropdown is opened.

Reply:
hmm yeah you're right. The definition of activated vs selected is confusing to me, but whatever spec says...